docs/PID tuning.md

```diff
@@ -4,30 +4,70 @@ Every aspect of flight dynamics is controlled by the selected "PID controller".
 responsible for reacting to your stick inputs and keeping the craft stable in the air by using the gyroscopes and/or
 accelerometers (depending on your flight mode).
 
-The "PIDs" are a set of tuning parameters which control the operation of the PID controller. The optimal PID settings
+The "PID gains" are a set of tuning parameters which affect the operation of the PID controller. The optimal PID settings
 to use are different on every craft, so if you can't find someone with your exact setup who will share their settings
-with you, some trial and error is required to find the best performing PID settings.
+with you, some trial and error is required to find the best performing PID settings. PID settings are also somewhat dependent on a pilot's preferences, so even if someone else has your exact same craft, you may prefer slightly different PID settings than them.
 
-A video on how to recognise and correct different flight problems caused by PID settings is available here:
+A video on how to recognise and correct different flight problems caused by PID settings is available [here](https://www.youtube.com/watch?v=YNzqTGEl2xQ). A detailed thread on tuning a mini-quad for acro-style FPV flight is [here](http://www.rcgroups.com/forums/showthread.php?t=2439428).
```

Comment:
More or less, since the same guy wrote it. I tried to keep the level of detail lower for the CF docs, and people who want to do a deeper dive can go to the RCGroups thread.